### DISCUSSION TITLE
What about working together?

### DISCUSSION THREAD

**Discussion by iddm** [jonasbb/serde_with#13] [Q&A]
I have created [`serde-aux crate`](https://github.com/vityafx/serde-aux) recently. We both follow the same ideas of implementing useful stuff which might be helpful for people using serde. But people now need different crates for some stuff, mine or yours. I think it would be better if we worked together on a single crate so we could provide with all-in-one helpful crate.

**Comment by jonasbb** [comment]
Hi, I wasn't aware of your crate when I started using this crate. I'm also somewhat surprised that people are sending PRs.

I agree that providing a single crate and working together will be beneficial.

**Comment by iddm** [comment]
Could we talk anywhere? discord, telegram, irc, anywhere?
